src/Host/Client/Impl/Host/RHost.cs

```diff
@@ -401,6 +401,15 @@ public sealed partial class RHost : IDisposable, IRExpressionEvaluator {
                                 _callbacks.DirectoryChanged();
                                 break;
 
+                            case "library":
+                                _callbacks.ViewLibrary();
```

Comment:
I have long wanted to get rid of all this hardcoded stuff in the host and in `IRCallbacks`, and instead offer a generic facility to register custom message callbacks by name.